app/controllers/two_factor_authentication/otp_verification_controller.rb

```diff
@@ -4,7 +4,10 @@ class OtpVerificationController < DeviseController
 
     def show
       assign_variables_for_otp_verification_show_view
-      analytics.track_event(Analytics::MULTI_FACTOR_AUTH_ENTER_OTP_VISIT, analytics_properties)
+      analytics.track_event(
+        Analytics::MULTI_FACTOR_AUTH_ENTER_OTP_VISIT,
+        analytics_properties.except(:confirmation_for_phone_change)
```

Comment:
what is the purpose of removing the `confirmation_for_phone_change` key here? based on the method name it looks like the hash is built specifically for analytics, so it is confusing as a reader to me to remove it?